packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -644,11 +644,11 @@ export class MgtPersonCard extends MgtTemplatedComponent {
    */
   protected renderPersonSubtitle(person?: IDynamicPerson): TemplateResult {
     person = person || this.internalPersonDetails;
-    if (!person.department) {
+    if (!(person as Person).department) {
```

Comment:
Let's avoid the casts here
```suggestion
    if (!isUser(person) || !person.department) {
```